assembly/std/coins/index.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * This namespace is used to interact with wallet's **balance** and **transfers**.
+ * It is located at **std/coins**.
```

Comment:
I think we should mathc what the dev when loading the library is seeing instead of the location of the file.

hence I propose:
```suggestion
 * It is located at **coins**.
```